.github/workflows/build_wheels.yml

```diff
@@ -3,10 +3,11 @@ name: Build Python Wheels
 on:
   # allow manual runs
   workflow_dispatch:
+  
   # run when we tag a release
-  release:
-    types:
-      - "created"
+  #release:
+  #  types:
+  #    - "created"
```

Comment:
Why is this being commented out?

Reply:
In practice, we've been including the wheels as part of a release vote. That means they need to be generated prior to creating a release tag, so anything automatically generated here wouldn't be used.

Maybe if someone downloaded the archive to extract the one they need manually as opposed to using pip install? But at that point it seems like they'd just as likely build a wheel locally.